addons/website/static/src/js/menu/content.js

```diff
@@ -3,13 +3,216 @@ odoo.define('website.contentMenu', function (require) {
 
 var core = require('web.core');
 var Dialog = require('web.Dialog');
+var time = require('web.time');
 var weContext = require('web_editor.context');
 var widget = require('web_editor.widget');
 var websiteNavbarData = require('website.navbar');
+var websiteRootData = require('website.WebsiteRoot');
+var Widget = require('web.Widget');
 
 var _t = core._t;
 var qweb = core.qweb;
 
+var PagePropertiesDialog = widget.Dialog.extend({
+    template: 'website.pagesMenu.page_info',
+    xmlDependencies: widget.Dialog.prototype.xmlDependencies.concat(
+        ['/website/static/src/xml/website.pageProperties.xml']
+    ),
+    events: _.extend({}, widget.Dialog.prototype.events, {
+        'click a.js_clone_page': '_onClonePageButtonClick',
+        'click a.js_delete_page': '_onDeletePageButtonClick',
+    }),
+    /**
+     * @constructor
+     * @override
+     */
+    init: function (parent, page_id, options) {
+        var self = this;
+        var server_url = window.location.origin;
+        var length_url = server_url.length;
+        var server_url_trunc = server_url;
+        if(length_url > 30)
+            server_url_trunc = server_url.slice(0,14) + '..' + server_url.slice(length_url-14,length_url);
+        self.server_url = server_url; 
```

Comment:
- camelCase variables (https://github.com/odoo/odoo/wiki/Javascript-coding-guidelines)
- Do not use `self` when you can use `this`
- `if () {`